ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 // ----- Utilities -----
-const removeNonNumeric = function(s) {
+const removeNonNumeric = function (s) {
```

Comment:
Truly speaking these can be:
```suggestion
function removeNonNumeric(s) {
```
I'll do a follow up clean up.